### PR TITLE
Sanitätsdienst content expansion

### LINKED ISSUE #46: Sanitätsdienst section pages' links are all dead
This is because I have not quite figuerd out the new distribution of contents after the rewrite.
I will fix that soon.

### PR DESCRIPTION
**Pull request by pantoffelemmy**
closes #46

### CODE REVIEW

**Review by LeonTReal — CHANGES_REQUESTED** [review]
Some edits I would discuss/change before merging:

**Page Erste Hilfe**
- Typo: Section _Sicherung von Verletzten_. Penultimate word "vorgenangen"
- Content change: Section _Medical Menu_. It is being suggested to rebind the default throw-grenade-keybind to `Shift + G`, but I'd rather suggest to simply unbind `G` for (vanilla) grenade throwing, since `Shift + G` is the default ACE Enhanced Grenade Throwing keybind
- Content addition: Section _Medizinische Versorgung_ >> _Blutvolumen_. Add explanation for word "Zugang"

**Page Behandlungsleitfaden**
- Typo: Section _Wundversorgung_ >> _second point_. Make "Nähen" lowercase
- Typo: Section _Blutkreislauf_ >> _second point_. "Debibrillator" to "Defibrillator"
- Typo: Section _Nachversorgung_ >> _third point_. "Split" to "Splint"

**Page Blutkreislauf**
- Typo: _Top paragraph_. "Bilden" must be lowercase
- Typo: Section _Wundversorgung_ >> _first paragraph_. "Im allgemeinen" to "Im Allgemeinen"
- Incorrect link: Section _Wundversorgung_ >> _Hämostase_ >> _fourth paragraph_. The word "Zugang" links to https://deploy-preview-57--tfhcwiki.netlify.app/articles/sanitaetsdienst/blutkreislauf#ausgleich-von-blutverlust, however the correct link is https://deploy-preview-57--tfhcwiki.netlify.app/articles/sanitaetsdienst/blutkreislauf#ausgleich-von-blutverlust-und-zug%C3%A4nge
- Typo: Section _Wundversorgung_ >> _Hämostase_ >> _fourth paragraph_. "Das bedeuetet, das ..." to "Das bedeutet, dass ..."
- Typo: Section _Ausgleich von Blutverlust und Zugänge_ >> _first paragraph_. "vorausgetzt" to "vorausgesetzt"
- Missing content: Section _Ausgleich von Blutverlust und Zugänge_ >> _end of first paragraph_. "Flüssigkeit verabreicht werden,"

**Page Medikamente**
- Dead link & missing content: Section _Rangfolge der Schmerzmedikamente_. The word "Fentanyl" has a dead links. **Also, the page is completely missing content about Fentanyl in general!**

**Page Sanitäter**
- Unfinished sentence: Section _Fähigkeiten_ >> _first sentence_. Sentence seems weird
- Typo: Section _Ausrüstung_ >> _Tourniquet_. "essentiell" to "essenziell"
- Typo: Section _Ausrüstung_ >> _Accuvac_ "Erbrochenen" to "Erbrochenem"
- Content change: _Nützliche Hinweise_ >> second point. Same as on page Erste Hilfe. Rather suggest to unbind `G`, since `Shift + G` is already default ACE Enhanced Grenade Throwing

I like all the new content, especially the new structure of Erste Hilfe. All pages were nice to read and easy to understand. If above changes have been discussed/implemented, the contents can be merged into main.

File: src/content/sanitaetsdienst/sanitaeter.svx [hunk 1]
```diff
@@ -42,39 +31,49 @@ Konkret können sie:
...
+ein großes Pflaster mit einem Ein-Weg-Ventil. Das Ventil lässt Luft und Flüssigkeiten aus dem Brustkorb entweichen und verhindert so einen Pneumothorax, siehe [Behandlung von Lungenverletzungen](/articles/sanitaetsdienst/atmung#lungen).
 
 #### 16g IV
 
 eine 16g-Kanüle (Durchmesser 1,5 mm) für intravenöse (IV) Zugänge. Wird benötigt, um Flüssigkeiten wie Blutinfusionen, aber auch manche Medikationen zu verabreichen. Diese Nadeln können nicht wiederverwendet werden, wie [in diesem YouTube-Video](https://youtu.be/rzLuZWWZ2eQ) anschaulich gezeigt wird.
 
 #### FAST IO
 
-_First Access for Shock and Trauma IO,_ ein intraossärer (IO) Zugang am Brustbein. Wird verwendet, wenn die Extremitäten für einen intravenösen Zugang zu stark verletzt sind.
+_First Access for Shock and Trauma IO,_ ein intraossärer (IO) Zugang am Brustbein. Wird verwendet, wenn die Extremitäten für einen intravenösen Zugang zu stark verletzt sind. Außerdem ist kein Puls nötig, um über diesen Zugang Flüssigkeiten verabreichen zu können<sup>[aktuell?]</sup>.
```
Comment (pantoffelemmy) [inline src/content/sanitaetsdienst/sanitaeter.svx:58]: @LeonTReal I once read that FAST IOs can do that but I can't recall where and have never tried it out, we should probably confirm that.

Comment (LeonTReal) [reply src/content/sanitaetsdienst/sanitaeter.svx:58]: Tested in game. FAST IOs allow fluids to flow into the body without a heartbeat.

**Comment by LeonTReal** [comment]
Reviewed all changes since my first comment.

Despite one typo I found, everything is perfect. When the typo has been fixed, I fully approve merging this PR.

Typo can be found here (sorry, currently using mobile):



"lösen" needs to be "löschen"

**Pull request merged**